packages/react-native-web/src/modules/createDOMProps/__tests__/index-test.js

```diff
@@ -140,6 +140,112 @@ describe('modules/createDOMProps', () => {
     });
   });
 
+  describe('onClick handlers', () => {
+    const callsOnClick = (component, accessibilityRole, disabled = false) => {
+      const onClick = jest.fn();
+      const event = { stopPropagation: jest.fn() };
+      const finalProps = createDOMProps(component, { accessibilityRole, disabled, onClick });
+      finalProps.onClick(event);
+      return onClick.mock.calls.length === 1;
+    };
+
+    test('are passed through and called for various roles', () => {
+      expect(callsOnClick('div', 'link')).toBe(true);
+      expect(callsOnClick('div', 'button')).toBe(true);
+      expect(callsOnClick('div', 'textbox')).toBe(true);
+      expect(callsOnClick('div', 'menuitem')).toBe(true);
+      expect(callsOnClick('div', 'bogus')).toBe(true);
+      expect(callsOnClick('a')).toBe(true);
+      expect(callsOnClick('button')).toBe(true);
+      expect(callsOnClick('input')).toBe(true);
+      expect(callsOnClick('select')).toBe(true);
+      expect(callsOnClick('textarea')).toBe(true);
+      expect(callsOnClick('h1')).toBe(true);
+    });
+
+    test('are not passed through or called for various roles when disabled is true', () => {
+      expect(callsOnClick('div', 'link', true)).toBe(false);
+      expect(callsOnClick('div', 'button', true)).toBe(false);
+      expect(callsOnClick('div', 'menuitem', true)).toBe(false);
+      expect(callsOnClick('a', undefined, true)).toBe(false);
+      expect(callsOnClick('button', undefined, true)).toBe(false);
+      expect(callsOnClick('input', undefined, true)).toBe(false);
+      expect(callsOnClick('select', undefined, true)).toBe(false);
+      expect(callsOnClick('textarea', undefined, true)).toBe(false);
+
+      expect(callsOnClick('div', 'textbox', true)).toBe(true);
+      expect(callsOnClick('div', 'bogus', true)).toBe(true);
+      expect(callsOnClick('h1', undefined, true)).toBe(true);
+    });
+  });
+
+  describe('keydown handlers', () => {
+    // These tests fire two events, one space and one enter to ensure each is handled correctly
+    const callsOnClick = key => (
+      component,
+      accessibilityRole,
+      disabled = false,
+      callOnSpace = false
+    ) => {
+      const onClick = jest.fn();
+      const onKeyDown = jest.fn();
+      const event = { key: 'Enter' };
```

Comment:
Just noting that I changed this in the merged patch so that `' '` was used for the spacebar tests